hal/src/electron/modem/mdm_hal.h

```diff
@@ -557,6 +558,7 @@ class MDMParser
     NetStatus   _net; //!< collected network information
     MDM_IP       _ip;  //!< assigned ip address
     MDM_DataUsage _data_usage; //!< collected data usage information
+    MDM_EdrxActs _edrxActs; //!< detected eDRX AcTs that are enabled
```

Comment:
We can store this structure on the stack and save some memory:
```
MDM_EdrxActs edrxActs = {};
if (RESP_ERROR == waitFinalResp(_cbCEDRXS, &edrxActs)) {
    ...
}
```

Reply:
True, I can replace the call to clear .count with just creating a new object which self memsets to 0 :)